tests/Core/_ToDo/MenuButton/FluentMenuButtonTests.cs

```diff
@@ -30,6 +31,7 @@ public void FluentMenuButton_Default()
             .Add(p => p.MenuStyle, menuStyle)
             .Add(p => p.Items, items)
             .Add(p => p.OnMenuChanged, onMenuChanged)
+            .Add(p => p.IconStart, SampleIcons.Info)
```

Comment:
Your unit test fails

Reply:
Hmm, all unit tests are passing for me locally running `dotnet test`. Is there more required to run unit tests locally?

Reply:
Sorry, just realized I forgot to add the updated "verified" file to my commit; the file has been added and pushed!